onnxruntime/contrib_ops/cpu/aten_ops/aten_op.h

```diff
@@ -22,8 +22,10 @@ class ATen : public OpKernel {
   std::string overload_name_;
 };
 
+#ifdef ENABLE_TRAINING
```

Comment:
These 2 funcs are used to fallback the ReduceSum Op to use libtorch by calling ATenExecutor directly instead of using the ATen Op. It's for ORTModule only, we don't want to fallback the ReduceSum for Inference. Previously all the ATen related code were under ENABLE_TRAING, so no macro was needed, when we move them for inference, since this 2 funcs are for ORTModule only, we need the ENABLE_TRAING macro here.